main.go

```diff
@@ -181,18 +225,24 @@ func processIQConfig(config configuration.IqConfiguration) {
 	if config.Application == "" {
 		LogLady.Info("No application specified, printing usage and exiting clean")
```

Comment:
Should a missing "application" arg really exist "clean"? Feels like maybe it should be an error code return value? (Which I added below, but I just noticed the log comment above).

Reply:
I think you are right, feel free to go with what makes sense!

Reply:
Done f369b29